### PR TITLE
Show a note about EULA where relevant

### PR DESCRIPTION
**Pull request by M4rtinK**
Show a note about EULA at the end of the installation
in GUI & TUI if the currently active install class
specifies a path to an EULA file.

If no EULA file path is specified, no EULA note will
be shown.

### CODE REVIEW

File: pyanaconda/ui/tui/spokes/installation_progress.py [hunk 1]
```diff
@@ -131,14 +131,22 @@ def show_all(self):
 
         util.ipmi_report(IPMI_FINISHED)
 
+        if self.instclass.eula_path is not None:
```
Comment (poncovka) [inline pyanaconda/ui/tui/spokes/installation_progress.py]: Please, remove `is not None`, so it is consistent with `pyanaconda/ui/gui/hubs/progress.py`.

File: pyanaconda/ui/tui/spokes/installation_progress.py [hunk 2]
```diff
@@ -131,14 +131,22 @@ def show_all(self):
...
+
         # kickstart install, continue automatically if reboot or shutdown selected
         if flags.automatedInstall and self.data.reboot.action in [KS_REBOOT, KS_SHUTDOWN]:
             # Just pretend like we got input, and our input doesn't care
             # what it gets, it just quits.
             raise ExitMainLoop()
 
     def prompt(self, args=None):
-        return Prompt(_("Installation complete. Press %s to quit") % Prompt.ENTER)
+        return Prompt(_("Press %s to quit") % Prompt.ENTER)
```
Comment (poncovka) [inline pyanaconda/ui/tui/spokes/installation_progress.py]: It looks like users will not see the `Installation complete.` message, if there is no EULA. Is it ok?

Comment (M4rtinK) [reply pyanaconda/ui/tui/spokes/installation_progress.py]: That looks like some sort of cherry-pick artifacts, I'll drop it.

File: pyanaconda/ui/tui/spokes/installation_progress.py (hunk 1 above)
Comment (M4rtinK) [reply pyanaconda/ui/tui/spokes/installation_progress.py]: Fixed.

**Pull request merged**